android/app/src/main/java/org/jitsi/meet/AudioModePackage.java

```diff
@@ -0,0 +1,32 @@
+package org.jitsi.meet.audiomode;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.JavaScriptModule;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class AudioModePackage implements ReactPackage {
+
+    @Override
+    public List<NativeModule> createNativeModules(
+                                ReactApplicationContext reactContext) {
+        List<NativeModule> modules = new ArrayList<>();
+        modules.add(new AudioModeModule(reactContext));
+        return modules;
+    }
+
+    @Override
+    public List<Class<? extends JavaScriptModule>> createJSModules() {
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+        return Collections.emptyList();
+    }
+}
```

Comment:
Even if I'm not bothering with documentation (javadocs/jsdocs-like) comments in the Objective-C source code, it would still be cool to have them everywhere like the coding style requires. For interface implementations, an `@inheritDoc` will be a very simple way to comply.